README.md

```diff
@@ -86,7 +106,7 @@ $ docker build -t moboff .
 $ xhost +
 
 # run the container
-$ docker run --it -e DISPLAY=$DISPLAY \
+$ docker run -it -e DISPLAY=$DISPLAY \
              -v /tmp/.X11-unix:/tmp/.X11-unix \
              moboff
```

Comment:
Can you also please add the link to the dockerhub page which will be a good reference? I was also thinking, what if we can add 'Dockerhub automatic build'. Once we change the Dockerfile and commit to github, an image gets built automatically. I guess I will add it as a new issue. What do you think about this @ashwani99 ?

Reply:
@rajula96reddy Yes, nice idea. Please create a new issue for it

Reply:
Thanks. I will create an issue.